tpl/tplimpl/embedded/templates/opengraph.html

```diff
@@ -2,9 +2,18 @@
 <meta property="og:description" content="{{ with .Description }}{{ . }}{{ else }}{{if .IsPage}}{{ .Summary }}{{ else }}{{ with .Site.Params.description }}{{ . }}{{ end }}{{ end }}{{ end }}" />
 <meta property="og:type" content="{{ if .IsPage }}article{{ else }}website{{ end }}" />
 <meta property="og:url" content="{{ .Permalink }}" />
-{{ with $.Param "images" }}{{ range first 6 . }}
+{{ with $.Params.images }}{{ range first 6 . -}}
```

Comment:
The above isn't correct (or: it's a breaking change that is not related to this)

Reply:
Other than that this looks great. Could you run `mage generate` to regenerate the Go code from the templates=